src/pages/groupComparison/Survival.tsx

```diff
@@ -75,7 +79,7 @@ export default class Survival extends React.Component<ISurvivalProps, {}> {
                             name,
                             color: blendColors(partitionGroupUids.map(uid => uidToGroup[uid].color)),
                             value,
-                            legendText: name
+                            legendText: JSON.stringify(partitionGroupUids),
```

Comment:
just want to confirm this is intended

Reply:
yes its a bit weird but that's how I'm passing it through the system I have rigged up